paella-matterhorn/plugins/edu.harvard.dce.infoButtonPlugin/dce_infobutton.js

```diff
@@ -14,18 +14,28 @@ Class ('paella.plugins.InfoPlugin', paella.ButtonPlugin,{
     var thisClass = this;
 
     var popUp = jQuery('<div id="dce-info-popup"></div>');
-    var buttonActions =[ 'Help with this player', 'Report a problem', 'Feedback', 'All Course Videos' ];
+    var buttonActions =[ 'Help with this player', 'Report a problem', 'DCE system status', 'Feedback', 'All Course Videos' ];
 
+    jQuery(popUp).append(thisClass.getItemTitle());
     buttonActions.forEach(function(item){
       jQuery(popUp).append(thisClass.getItemButton(item));
     });
     jQuery(domElement).append(popUp);
   },
 
+  getItemTitle: function () {
+    var mpInfo = paella.matterhorn.episode.mediapackage;
+    var titleDiv = mpInfo.title?'<span>' + mpInfo.title + '</span>':'';
```

Comment:
I'm a fan of ternary statements, but some whitespace here would be nice. I frequently even go overboard and format like:

    var titleDiv = mpInfo.title
      ? '<span>' + mpInfo.title + '</span>'
      : '';